contents/blog/best-launchdarkly-alternatives.mdx

```diff
@@ -0,0 +1,496 @@
+
+- 🤝 **Integrations:** Combine with the tools you are already using for your workflow like GitHub, Terraform, Datadog, and Jira.
+
+- 🧪 **A/B testing:** Run experiments and evaluate the impact of new features. Do multi-armed bandits to compare variations.
+
+- 🧑‍💻 **Developer-focused:** Features like a server-less architecture, edge decisioning support, and a wide API help developers make full use of DevCycle.
+
+### How does DevCycle compare to LaunchDarkly?
+
+DevCycle has many of the core A/B testing and feature flagging features but misses out on some of the more advanced features of LaunchDarkly.
+
+<ComparisonTable column1="DevCycle" column2="LaunchDarkly">
+  <ComparisonRow column1={true} column2={false} feature="Self-serve" description="Free to try, no mandatory sales calls" />
+  <ComparisonRow column1={true} column2={true} feature="Feature flags" description="Deploy features safely with targeting and percentage rollouts" />
+  <ComparisonRow column1={"Segments"} column2={true} feature="Advanced targeting" description="Target flags and A/B tests on segments and non-users" />
+  <ComparisonRow column1={true} column2={true} feature="Local evaluation" description="Use local, cached flag values to increase speed" />
+  <ComparisonRow column1={true} column2={true} feature="A/B testing" description="Run multivariate tests and see the impact of changes with custom goals and reports" />
+  <ComparisonRow column1={false} column2={true} feature="Funnel tests" description="Use conversion funnels as goals for your A/B test" />
+  <ComparisonRow column1={false} column2={false} feature="Advanced analytics" description="Custom SQL, paths, and retention analysis with flag data and beyond." />
+  <ComparisonRow column1={true} column2={true} feature="Major SDKs" description="Pre-built SDKs for JavaScript, Python, iOS, Android, Go, and more" />
+  <ComparisonRow column1={"Edit only"} column2={true} feature="API" description="Evaluate and edit flags and A/B tests with an API" />
+  <ComparisonRow column1={true} column2={true} feature="Automations" description="Trigger flag states based on schedules, triggers" />
+  <ComparisonRow column1={"Exports only"} column2={true} feature="Imports and exports" description="Import and export data to and from other popular tools" />
+  <ComparisonRow column1={"No audit log"} column2={true} feature="Governance" description="Audit log, roles-based access, permissions" />
+  <ComparisonRow column1={"SDKs"} column2={"SDKs"} feature="Open source" description="Audit code, contribute to roadmap, and build integrations" />
+</ComparisonTable>
+
+### How popular is DevCycle?
+
+BuiltWith doesn't have data on DevCycle's usage, but we can use organic traffic as a proxy. Ahrefs estimates its organic traffic to be 3.5k per month, while it estimates LaunchDarkly at 46.3k per month, nearly 15x more popular.
```

Comment:
I think Google trends might be a better alternative for this: https://trends.google.com/trends/explore?date=today%205-y&q=devcycle,launchdarkly,posthog

Come to think of it, maybe we should add this going forward?

In any case, Ahrefs measures traffic to any content, whereas as Trends will show searches for their brand name in Google.

Reply:
BuiltWith feels more tangible, so I think we should try to use it if possible. For some (like LaunchDarkly), it might be better to use Trends because they don't get captured well by however BuiltWith measures tools.